src/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp

```diff
@@ -33,6 +33,12 @@ std::vector<ov::element::Type> prcs = {
     ov::element::u64,
 };
 
+std::vector<ov::element::Type> input_prcs = {
```

Comment:
Please add all possible precisions, like here: https://github.com/openvinotoolkit/openvino/pull/9667/files#diff-8d7290423c9eb94aa21ff3b7b88f3c750877f34d1015b78b706db5c97db7d76aR22

Reply:
Enabled old precisions but enabled logic to skip unsupported input precisions. So, if we add the new supported precision future the test will be failed